lib/index.js

```diff
@@ -0,0 +1,323 @@
+/*!
+ * Copyright (c) 2018 Digital Bazaar, Inc. All rights reserved.
+ */
+const async = require('async');
```

Comment:
Don't use `async` in new modules.

Reply:
From copy & paste to get things rolling.  Will refactor it away moving forward.